nuxeo-management-rest-api/src/test/resources/OSGI-INF/elasticsearch-test-contrib.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+
+<component name="org.nuxeo.elasticsearch.test.contrib">
```

Comment:
If you need to define a test component here, please use a different name form the one in nuxeo-elasticsearch-core.
In fact, you might be able to deploy `org.nuxeo.elasticsearch.core.test:elasticsearch-test-contrib.xml`.

Reply:
go for a new component, i agree ;)

Reply:
Why not try to reuse the existing one?

Reply:
you right this file was added before the pom modification (to include the test jar dependency) and i forget about removing the new file and use the core one. thx

Reply:
Agree, prefix the component name with something common to the module, such as `org.nuxeo.rest.management`.
`org.nuxeo.rest.management.elasticsearch.test`?